site/src/_layouts/component.hbs

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+---
+<div class="container component">
+
+  <div class="row">
+
+    <div class="col-sm-9 col-xs-12">
+      <h1>
+        {{title}}
+        <a href="{{package.repository.url}}/blob/master/src/{{component}}">View source on GitHub</a>
+      </h1>
+      {{#markdown }}
+        {{> body  }}
+      {{/markdown }}
+    </div>
+
+    <nav class="col-sm-3 hidden-xs">
```

Comment:
I think this is covered by the task for adding next/previous buttons to the components. It's not ideal but a bit of a compromise due to assemble not supporting (in a nice way) including rendered content.